setup.py

```diff
@@ -41,7 +42,19 @@ def deduce_version():
 
 
 setup(
-    name="pyoorb",
+    name='pyoorb',
+    maintainer="oorb developers",
+    maintainer_email="oorb@googlegroups.com",
+    description="An open-source orbit-computation package for Solar System objects. ",
+    long_description=Path("README.md").read_text(encoding="utf-8"),
+    long_description_content_type="text/markdown",
+    url="https://github.com/oorb/oorb",
+    author="Mikael Granvik et al.",
+    download_url="https://pypi.python.org/pypi/oorb",
```

Comment:
Actually, here's a small nitpick - maybe we want to call it pyoorb in pypi which would make https://pypi.python.org/pypi/pyoorb the proper `download_url` here. My fault for suggesting this one earlier!

Reply:
Oops, good catch. Fixed.